mad_spark_multiagent/web/frontend/src/components/ResultsDisplay.tsx

```
@@ -233,21 +361,27 @@ const ResultsDisplay: React.FC<ResultsDisplayProps> = ({ results }) => {
 
             {/* Action Buttons */}
             <div className="mt-4 flex space-x-3">
-              <button className="inline-flex items-center px-3 py-2 border border-gray-300 shadow-sm text-sm leading-4 font-medium rounded-md text-gray-700 bg-white hover:bg-gray-50 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-blue-500">
-                <svg className="h-4 w-4 mr-1.5" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+              <button 
+                onClick={() => handleBookmark(index)}
+                className={`inline-flex items-center px-3 py-2 border ${bookmarkedIdeas.has(index) ? 'border-blue-500 bg-blue-50' : 'border-gray-300'} shadow-sm text-sm leading-4 font-medium rounded-md ${bookmarkedIdeas.has(index) ? 'text-blue-700' : 'text-gray-700'} bg-white hover:bg-gray-50 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-blue-500`}>
+                <svg className="h-4 w-4 mr-1.5" fill={bookmarkedIdeas.has(index) ? 'currentColor' : 'none'} viewBox="0 0 24 24" stroke="currentColor">
                   <path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2} d="M5 5a2 2 0 012-2h10a2 2 0 012 2v16l-7-3.5L5 21V5z" />
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)